repos/tap/components/form/select/select.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import { 
+  Select as KegSelect, 
+  Option, 
+  Label,
+  View
+} from '@keg-hub/keg-components'
+import { reStyle } from '@keg-hub/re-theme/reStyle'
+
+
+const ReStyleView = reStyle(
+  View,
+  'style'
+)((__, props) => ({
+  ...props?.style,
+  padding: 8,
+}))
```

Comment:
Sweet. I really like this way of styling components. I like that it separates all the styling logic into a function completely separate from the component, and it still has access to the props for dynamic stuff.

Reply:
Yah, ReStyle is the future